packages/core/admin/admin/src/pages/Admin/Onboarding/index.js

```diff
@@ -1,59 +1,82 @@
-import React, { useState } from 'react';
+import React, { useRef, useState } from 'react';
 import styled from 'styled-components';
 import { useIntl } from 'react-intl';
-import { Box, Flex, FocusTrap, Typography, Icon, Stack } from '@strapi/design-system';
-import { Book, Cross, Information, Question } from '@strapi/icons';
-import { pxToRem } from '@strapi/helper-plugin';
+import {
+  Box,
+  Button,
+  Divider,
+  Flex,
+  FocusTrap,
+  Icon,
+  Portal,
+  PopoverPrimitives,
+  Stack,
+  Typography,
+  VisuallyHidden,
+} from '@strapi/design-system';
+import { Cross, Play, Question } from '@strapi/icons';
 
 import { useConfigurations } from '../../../hooks';
+import onboardingPreview from '../../../assets/images/onboarding-preview.png';
```

Comment:
I have no idea about this, but do we _have_ to bundle this? we dont export our own `public` images?

Reply:
to be honest I don't know - but we have some others in here 
<img width="327" alt="image" src="https://user-images.githubusercontent.com/71838159/217483034-3c732037-367c-48c8-849d-3760b8986b23.png">

Reply:
@gu-stav do you have any knowledge about this topic? 👀